ui/web/src/styles/tokens.css

```
@@ -0,0 +1,9 @@
+:root {
+  --bg: #f7f8fa; --surface: #ffffff; --border: #e2e5ea;
+  --text: #1b1f24; --text-muted: #5c6470;
+  --accent: #2f6df0; --accent-text: #ffffff;
+  --ok: #1a7f4b; --warn: #b8860b; --error: #c0392b;
+  --radius: 6px; --space-1: 4px; --space-2: 8px; --space-3: 16px; --space-4: 24px;
+  --font: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
```

Stylelint (17.13.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)